client/app/views/konnector.coffee

```diff
@@ -303,7 +303,7 @@ class="folder-link"
 target="_blank">
 #{t "open selected folder"}
 </a>
-"""
+""" if 'ending' in (window.user.onboardedSteps or [])
```

Comment:
Oops. It seems that I gave you the wrong step name. I am really sorry about that. The good one is `confirmation`. That's the name of the last step. `ending` was the last one in a previous server version.